pkg/node/node.go

```diff
@@ -454,7 +454,7 @@ func NewBee(addr string, swarmAddress swarm.Address, publicKey ecdsa.PublicKey,
 
 	traversalService := traversal.NewService(ns)
 
-	pushSyncProtocol := pushsync.New(p2ps, storer, kad, tagService, pssService.TryUnwrap, logger, acc, accounting.NewFixedPricer(swarmAddress, 1000000000), tracer)
+	pushSyncProtocol := pushsync.New(p2ps, storer, kad, tagService, pssService.TryUnwrap, postage.AttachValidStamp(batchStore), logger, acc, accounting.NewFixedPricer(swarmAddress, 1000000000), tracer)
```

Comment:
Not sure why the protocol should care about postage stamp validity. The necessity of these changes is also not documented in the writeup

Reply:
push  synced chunks need  to have a valid stamp attached if you dont validate you risk being disconnected and blocklisted for invalid stamp